CHANGES.md

```diff
@@ -2,6 +2,12 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.427 Aug 06 2025
```

Comment:
While updating the changes - it would also make sense to fill up the gap for the other releases that we've missed.

Reply:
Can that be addressed as a follow-up MR? We'd like to move forward with having a new release with other changes

Reply:
It is more we would like to avoid investing the time on gathering the conceptual changes right now

Reply:
AI for the rescue? https://github.com/openshift-online/ocm-api-model/pull/1086